.github/workflows/e2e_test.yml

```diff
@@ -0,0 +1,42 @@
+name: E2E Tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      # trigger workflow if PR is opened directly as R4R.
+      - opened
+      # trigger workflow if changes are pushed to the branch.
+      - synchronize
+      # trigger workflow if PR is marked ready for review.
+      - ready_for_review
+    paths-ignore:
+      - 'docs/**'
+      - '**.md'
+      - 'LICENSE'
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:   
+  # e2e-tests-by runs the actual go test command to trigger the test.
+  e2e-tests:
+    strategy:
+      matrix:
+        tests: 
+          - "e2e-test-ibc"
+          - "e2e-test-ibc-timeout"
```

Comment:
Does this list need to be updated with every new test? Not the best UX

Reply:
That's right, we need to update it every time there's a new test added. I'm considering writing some logic to extract new tests and add them to the workflow. It might happen in the near future.

Reply:
Ok, let's consider that when our test base grows. Let's carry on for now